README.md

```diff
@@ -53,9 +65,217 @@ This is the simplest case. Get the script in whatever way you prefer and include
 </script>
 ```
 
-## License
+## Usage
+The Brightcove Player Loader exposes a single function. This function takes an parameters object which describes the player it should load and returns a `Promise` which resolves when the player is loaded and created and rejects if anything fails.
+
+### Examples
```

Comment:
After the plugin is release perhaps include a jsbin for this working end to end with a demo account?

Reply:
We could, but I think that we'd want proper documentation on our support site (with demos).